unity-client/Assets/Scripts/MainScripts/DCL/Controllers/PlayerAvatarController/PlayerAvatarController.cs

```diff
@@ -51,6 +51,11 @@ private void Update()
         avatarVisibility.SetVisibility("PLAYER_AVATAR_CONTROLLER", shouldBeVisible);
     }
 
+    public void SetAvatarVisibility(bool isVisible)
+    {
+        avatarRenderer.gameObject.SetActive(isVisible);
+    }
```

Comment:
Instead of this new method, we could remove it and you can call this object's existent `avatarRenderer.Setvisibility(bool)` that does the same thing, so that we have that functionality centralized in 1 place.